post/OpenSeesPost.dpr

```diff
@@ -3389,39 +3404,54 @@ begin
                            't1','t2','N/A','','','','kPa',4,5,true,3,4,-1,-1,-1,-1);
 
         WriteResults(false,
                      false,'Reading ShellDKGQ strains (membrane) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Membrane (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Membrane (L'+IntToStr(i)+')',
                            'e11','e22','e12','','','','-',4,5,true,0,1,2,-1,-1,-1);
 
         WriteResults(false,
                      false,'Reading ShellDKGQ strains (shear) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Shear (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Shear (L'+IntToStr(i)+')',
                            'g1','g2','N/A','','','','-',4,5,true,3,4,-1,-1,-1,-1);
 
+        WriteResults(false,
+                     false,'Reading ShellDKGQ Axial stresses for layer '+IntToStr(i)+' ',
+                           'ShellDKGQ_steel_stress_Layer'+IntToStr(i),'ShellDKGQ','Axial-Stresses (L'+IntToStr(i)+')',
+                           's11','','','','','','kPa',4,1,true,0,-1,-1,-1,-1,-1);
+
+        WriteResults(false,
+                     false,'Reading ShellDKGQ Axial strains for layer '+IntToStr(i)+' ',
+                           'ShellDKGQ_steel_strain_Layer'+IntToStr(i),'ShellDKGQ','Axial-Strains (L'+IntToStr(i)+')',
+                           'e11','','','','','','-',4,1,true,0,-1,-1,-1,-1,-1);
+
         WriteResults(false,
                       true,'Reading ShellDKGQ principal stresses (membrane) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_stress_Layer'+IntToStr(i),'ShellDKGQ','Stresses-Membrane-Principal (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_stress_Layer'+IntToStr(i),'ShellDKGQ','Stresses-Membrane-Principal (L'+IntToStr(i)+')',
                            's1','s2','angle','','','','kPa',4,5,true,0,1,2,-1,-1,-1);
 
         WriteResults(false,
                       true,'Reading ShellDKGQ principal strains (membrane) for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Membrane-Principal (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_strain_Layer'+IntToStr(i),'ShellDKGQ','Strains-Membrane-Principal (L'+IntToStr(i)+')',
                            'e1','e2','angle','','','','-',4,5,true,0,1,2,-1,-1,-1);
 
         WriteResults(false,
                      false,'Reading ShellDKGQ damage indices for layer '+IntToStr(i)+' ',
-                           'ShellDKGQ_TempKtKc_Layer'+IntToStr(i),'ShellDKGQ','KtKc (L'+IntToStr(i)+')',
+                           'ShellDKGQ_concrete_TempElong_KtKc_Layer'+IntToStr(i),'ShellDKGQ','KtKc (L'+IntToStr(i)+')',
```

Comment:
For consistency, kindly change the name of the GiD output from 'KtKc' to 'Concrete Temp & Damage':
so:
'ShellDKGQ_concrete_TempElong_KtKc_Layer'+IntToStr(i),'ShellDKGQ','KtKc (L'+IntToStr(i)+')',
becomes:
'ShellDKGQ_concrete_TempElong_KtKc_Layer'+IntToStr(i),'ShellDKGQ','Concrete Temp & Damage (L'+IntToStr(i)+')',

Reply:
Ok!